packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -169,9 +174,44 @@ - (instancetype)initWithCameraName:(NSString *)cameraName
   _motionManager = [[CMMotionManager alloc] init];
   [_motionManager startAccelerometerUpdates];
 
-  if (![self setCaptureSessionPreset:_resolutionPreset withError:error]) {
-    return nil;
+  NSError *outError = nil;
```

Comment:
Why is this scoped more broadly than the block using it?

Reply:
moved into block.